docs/getting_started/discover_ui.md

```diff
@@ -4,22 +4,190 @@ description: Take a quick tour of the Back Office.
+- access Structure view and use 'Move up' and 'Move down' function in the settings of the block
+
+When you add a new block to the drop zone, drop it in the blue highlighted area. Before you drop it, a bold line appears  - it helps you see the position of the newly added block in relation to other, already added blocks.
+
+![Drop zone line](img/dashboard_blue_line.png)
+
+When you add a block by dragging it from Elements menu into the drop zone,
+the block settings panel open immediately where you can configure all block properties.
+
+![Block properties](img/dashboard_block_properties.png)
+
+This is a default behavior. You can globally turn off automatic opening of the block settings panel in the user settings.
+First, access your user settings on the right side of the top bar:
+
+![User settings](img/user_settings.png "User settings")
+
+Then, go to **Preferences** tab, **Edit** section.
+Here, you can find `Automatically open block settings in builder` setting, which, by default, is set up to `enabled` value.
+To change this behavior, click on **Edit** icon, find the setting, and change its value to `disabled`.
+
+![Blocks settings](img/user_settings_blocks.png "User settings - blocks settings")
+
+#### Work with blocks
+
+Each block has its special properties.
+You can access them by placing the cursor on the added block and clicking the **Block settings** icon.
+
+![Block settings - Dashboard Builder](img/block_settings_builder.png "Block settings - Dashboard Builder")
+
+Settings available for blocks are divided into two tabs: **Properties** and **Design**.
+The settings available on the **Properties** tab are tailored to the block's content.
+For a description of these settings, see [Dashboard block reference](dashboard_block_reference.md).
+
+To save changes, click **Save and close**.
+
+#### Edit dashboard
+
+At any time you can edit an active customized dashboard.
+To do it, click an icon in the top right corner, and then click **Edit My dashboard**.
+
+![Edit dashboard](img/edit_active_dashboard.png "Edit dashboard")
+
+This action opens Dashboard Builder that allows you to edit an active customized dashboard and publish its updated version.
+You can see all the implemented changes after reloading starting screen.
+
+#### Select active dashboard
+
+When you create new customized dashboard, you can choose, which dashboard you want to set as an active.
+
+To do it, access your user settings on the right side of the top bar.
+Then, go to **Preferences** tab, **Dashboard** section, and click **Edit**.
+
+Select the dashboard, that you want to set as active, and click **Save**.
+
+![Select active dashboard](img/select_active_dashboard.png "Select active dashboard")
+
+#### Permissions
+
+In order to be able to customize dashboard, you need to have `dashboard/cutomize` policy.
+By default, all the users belonging to the `Editors` User Group, have `Dashboard` role assigned, so they can edit, create, or delete dashboard.
```

Comment:
```suggestion
By default, all users belonging to the `Editors` User Group have the `Dashboard` role assigned, so that they can edit, create, or delete dashboards.
```

Reply:
Dashboard, singular form here is correct.